java8-okhttp-gson/pom.xml

```diff
@@ -5,7 +5,7 @@
   <artifactId>blockchain-proof-sdk-java8-okhttp-gson</artifactId>
   <packaging>jar</packaging>
   <name>blockchain-proof-sdk-java8-okhttp-gson</name>
-  <version>0.10.0</version>
+  <version>0.10.4</version>
```

Comment:
Should this be changed in java8-okhttp-gson? Shouldn't this .pom be auto-generated from our buildAllMavenProfiles.sh?